pkg/reconciler/controllers/constants.go

```diff
@@ -12,19 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package reconcile
+package controllers
 
-import (
-	"os"
-	"testing"
-
-	"k8s.io/klog/v2"
+// reconciler controller names
+const (
+	// SyncerController is the name of the syncer controller running in the reconciler container.
+	// It is named syncer instead of parser because it includes parsing, validating and applying.
+	// Also, there is already a Parser struct declared in pkg/parse/opts.go.
+	SyncerController = "Syncer"
+	// RemediatorController is the name of the remediator controller running in the reconciler container.
+	RemediatorController = "Remediator"
```

Comment:
Suggest putting `const` where it's used instead of creating a const file for all constants.